feature-libs/pickup-in-store/components/container/my-preferred-store/my-preferred-store.component.ts

```diff
@@ -69,27 +89,75 @@ export class MyPreferredStoreComponent implements OnInit {
     );
 
     useFeatureStyles('a11yViewHoursButtonIconContrast');
+    useFeatureStyles('a11yImproveButtonsInCardComponent');
   }
 
   ngOnInit(): void {
-    this.cmsService
-      .getCurrentPage()
-      .pipe(
-        filter<Page>(Boolean),
-        take(1),
-        tap(
-          (cmsPage) =>
-            (this.isStoreFinder = cmsPage.pageId === 'storefinderPage')
-        ),
-        filter(() => this.isStoreFinder),
-        tap(() => {
-          this.content = {
-            header: '',
-            actions: [{ event: 'send', name: 'Get Directions' }],
-          };
-        })
-      )
-      .subscribe();
+    if (
+      this.featureConfigService.isEnabled('a11yImproveButtonsInCardComponent')
+    ) {
+      this.cmsService
+        .getCurrentPage()
+        .pipe(
+          filter<Page>(Boolean),
+          take(1),
+          map((cmsPage) => {
+            this.isStoreFinder = cmsPage.pageId === 'storefinderPage';
```

Comment:
Because we don't use the global `this.isStoreFinder` ourselves (unless it is used in the template?) and rather just the local `isStoreFinder`, lets just return the boolean instead of setting it globally too. I think the global variable should be removed if it doesn't really serve a purpose. Could we please also leave a `//TODO: Remove in major`-like comment if it isn't needed?